packages/nimble-components/docs/component-status.mdx

```diff
@@ -0,0 +1,16 @@
+import { Story, Meta } from '@storybook/addon-docs';
+import { componentStatus } from './component-status.stories';
+
+<Meta title="Components/Status Table" />
+
+# Component status
```

Comment:
Can we use a more consistent name? ie the tree says the title is Status Table while the header here is Component status.

Maybe if moved next to Getting Started in the TOC it could just be "Component Status"?
```
Getting Started
Component Status
COMPONENTS
  Anchor
  ...
```